tracer/src/Datadog.Trace/TraceTagNames.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="TraceTagNames.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace;
+
+/// <summary>
+/// Standard span tags used by integrations.
```

Comment:
Can you update the comment to refer to trace level tags?

Reply:
Oops. Good catch.

Reply:
Fixed in cb629482b84a29575065ad03403fc7c71741ba07 and e471b5336b3e9619511e8ce4e3fccc267d5db3a1.

```xml
/// <summary>
/// Names used for trace-level tags.
/// </summary>
```